docs/src/components/AsciinemaPlayer.tsx

```diff
@@ -26,18 +27,25 @@ const AsciinemaPlayerComponent: React.FC<AsciinemaPlayerProps> = ({
   const ref = useRef<HTMLDivElement>(null);
   const playerCreated = useRef(false);
   const proxiedSrc = `https://corsproxy.io/?${encodeURIComponent(src)}`;
+  const [isMounted, setIsMounted] = useState(false);
+  const [isPlayerLoaded, setIsPlayerLoaded] = useState(false);
```

Comment:
Isn't it sufficient to only add the `isMounted` state? Can't we remove the `isPlayerLoaded` state?

Reply:
I added `isPlayerLoaded` to validate if the player is in a ready state. If we don't add it, the player will flicker the first few milliseconds on load and the page looks a bit weird then.   

Although the added class doesn't do a lot yet, I just thought of a possible alternative to improve this initial flicker a bit more. I'll keep you posted.

Reply:
I'm keeping `isPlayerLoaded` because it serves a distinct role from mount in determining if the player is ready to be displayed.

However, since the method to reduce flicker on load didn't make much difference, I decided to remove that part of the logic.

Reply:
I don't really see a difference between `isMounted` and `isPlayerLoaded`. For me it seems to work perfectly, even if I remove the `isPlayerLoaded`.

Reply:
I noticed a bit of latency difference when implementing `isPlayerLoaded` with the flicker. But it's quite a minor difference so I can drop it to keep it more sustainable.